src/components/OurFilosofy.astro

```diff
@@ -0,0 +1,36 @@
+---
+import nimionLogo from '/public/images/nimion-primary-dark.png';
+import triangleBg from '/public/images/triback.png';
+---
+<section class="px-20 font-montserrat flex flex-col bg-primary-dark-background/95 bg-background-triangle bg-contain bg-no-repeat">
+    <div class="flex justify-between items-center mt-20">
+        <div>
+            <h4 class="font-semibold text-xl text-golden">Nuestra Filosofia de Trabajo</h4>
+
+            <h3 class="mt-10 font-bold text-4xl text-white">Un Producto Excepcional</h3>
+            <div class="w-144 mt-5 border-b-2 border-secondary-light"></div>
+        </div>
+        <img class="w-[4rem] h-[4rem]" src={nimionLogo} alt="nimion-logo">
+    </div>
+
+    <div class="flex justify-between items-center w-full py-10">
+        <div class="flex flex-col items-center justify-center">
+            <h4 class="px-28 text-center font-medium text-lg text-golden leading-10">"Queremos reivindicar el significado de la palabra excelencia"</h4>
+            <div class="w-20 mt-3 border-b-2 border-secondary-light"></div>
```

Comment:
Here I think you can put some weight on the font because it seems to disappear a bit, a suggestion

Reply:
it's truth. I'm on that